docs/core-concepts/dotnet-core.md

```diff
@@ -0,0 +1,145 @@
+
+- [ASP.NET Core](http://asp.net)
+- [Windows 10 Universal Windows Platform (UWP)](https://developer.microsoft.com/windows)
+- [Xamarin.Forms](https://www.xamarin.com/forms)
+
+Open Source
+-----------
+
+[.NET Core](https://github.com/dotnet/core) is open source (MIT license) and was contributed to the [.NET Foundation](http://dotnetfoundation.org) by Microsoft in 2014. It is now one of the most active .NET Foundation projects. It can be freely adopted by individuals and companies, including for personal, academic or commercial purposes. Multiple companies use .NET Core as part of apps, tools, new platforms and hosting services. Some of these companies make significant contributions to .NET Core on GitHub and provide guidance on the product direction as part of the [.NET Foundation Technical Steering Group](http://www.dotnetfoundation.org/blog/tsg-welcome).
+
+Acquisition
+===========
+
+.NET Core is distributed in two main ways, as packages on NuGet.org and as standalone distributions.
+
+Distributions
+-------------
+
+You can download .NET Core at the [.NET Core Getting Started](https://www.microsoft.com/net/core) page.
+
+- The *Microsoft .NET Core* distribution includes the CoreCLR runtime, associated libraries, a console application host and the `dotnet` app launcher. It is described by the [`Microsoft.NETCore.App`](https://www.nuget.org/packages/Microsoft.NETCore.App) metapackage.
+- The *Microsoft .NET Core SDK* distribution includes .NET Core and a set of tools for restoring NuGet packages and compiling and building apps. 
+
+Typically, you will first install the .NET Core SDK to get started with .NET Core development. You may choose to install additional .NET Core (perhaps pre-release) builds.
+
+Packages
+--------
+
+- [.NET Core Packages](packages.md) contain the .NET Core runtime and libraries (reference assemblies and implementations). For example, [System.Net.Http](https://www.nuget.org/packages/System.Net.Http/).
+- [.NET Core Metapackages](packages.md) describe various layers and app-models by referencing the appropriate set of versioned library packages.
+
+Architecture
+============
+
+.NET Core is a cross-platform .NET implementation. The primary architectural concerns unique to .NET Core are related to providing platform-specific implementations for supported platforms.
+
+Environments
+------------
+
+.NET Core is supported by Microsoft on Windows, OS X and Linux. On Linux, Microsoft primarily supports .NET Core running on Red Hat Enterprise Linux (RHEL) and Debian distribution families.
+
+.NET Core currently supports X64 CPUs. On Windows, X86 is also supported. ARM64 and ARM32 are in progress.
+
+The [.NET Core Roadmap](https://github.com/dotnet/core/blob/master/roadmap.md) provides more detailed information on workload and OS and CPU environment support and plans.
```

Comment:
The roadmap could use some fleshing out. It doesn't mention any future platforms, like ARM